--- server-side/fstab.mdkTS	2002-05-30 17:00:26.000000000 +0200
+++ server-side/fstab	2006-03-16 14:12:07.000000000 +0100
@@ -1,13 +1,16 @@
-# /etc/fstab for diskless boot clients
-# Save this file as /etc/fstab$$CLIENT$$
-# $Id: fstab,v 1.3 2002/05/30 15:00:26 mcb30 Exp $
-
-10.0.0.8:/ / nfs soft,ro,rsize=8192,wsize=8192 0 0
-none /proc proc defaults 0 0
 none /tmp tmpfs defaults 0 0
+none /var/lib/gnome/desktop tmpfs noauto,nodev,nosuid 0 0
+none /var/lib/xdm tmpfs defaults 0 0
+none /var/lib/alsa/dev tmpfs defaults 0 0
 none /var/log tmpfs defaults 0 0
 none /var/run tmpfs defaults 0 0
+none /var/tmp tmpfs defaults 0 0
 none /var/lock/subsys tmpfs defaults 0 0
+none /etc/sysconfig/harddrake2 tmpfs defaults 0 0
 none /root tmpfs defaults 0 0
-none /dev/shm tmpfs defaults 0 0
-none /dev/pts devpts mode=0620 0 0
+# basic cdrom/floppy mount points
+# multiple clients should probably have unique mount points
+# such as /var/lib/terminal-server/clients/client-ip-address/mnt/cdrom
+/dev/cdrom /mnt/cdrom iso9660 user,iocharset=iso8859-1,noauto,ro,exec 0 0
+none /mnt/floppy supermount dev=/dev/fd0,fs=ext2:vfat,--,umask=0,iocharset=iso8859-1,sync,codepage=850 0 0
+
--- server-side/inittab.mdkTS	2002-05-01 10:35:32.000000000 +0200
+++ server-side/inittab	2006-03-16 02:05:25.000000000 +0100
@@ -1,5 +1,4 @@
-# /etc/inittab$$CLIENT$$
-# $Id: inittab,v 1.1 2002/05/01 08:35:32 dan Exp $
+# /etc/inittab for terminal-server clients
 
 id:5:initdefault:
 
@@ -14,9 +13,6 @@
 l5:5:wait:/etc/rc.d/rc 5
 l6:6:wait:/etc/rc.d/rc 6
 
-# Things to run in every runlevel.
-ud::once:/sbin/update
-
 # Trap CTRL-ALT-DELETE
 ca::ctrlaltdel:/sbin/reboot -f
 
